ambry-utils/src/main/java/com.github.ambry.utils/Crc32.java

```diff
@@ -96,6 +97,51 @@ public void update(byte[] b, int off, int len) {
     crc = localCrc;
   }
 
+  public void update(ByteBuffer buffer) {
```

Comment:
`updateCrc`? or better a better name?

Reply:
I am using the same name for other types. But I updated the method name in the CrcInputStream.